commands/version/lib/git-repo-url.js

```diff
@@ -0,0 +1,16 @@
+"use strict";
+
+const log = require("libnpm/log");
+const childProcess = require("@lerna/child-process");
+
+module.exports = getRepoUrl;
+
+function getRepoUrl(opts) {
+  log.silly("getRepoUrl");
+
+  const args = ["config", "--get", "remote.origin.url"];
```

Comment:
You'll need to account for the [`--git-remote`](https://github.com/lerna/lerna/blob/c79a82722095607940749f5c6fa1cdd90c56a437/commands/version/command.js#L45-L50) option here.